style.css

```diff
@@ -137,4 +136,97 @@ nav ul {
     fill: var(--gray-2);
     top: 50%;
     transform: translateY(-50%);
+}
+
+.img-teams {
+    width: 8em;
+}
+
+/* main */
+
+main {
+    width: 60%;
+}
+
+h1 {
+    font-size: 25px;
+    font-weight: normal;
+}
+
+.btn-gray.right-radius {
+    border-bottom-right-radius: 0.2em;
+    border-top-right-radius: 0.2em;
+}
+
+.filter-question .btn-gray {
+    border-radius: 0;
+}
+
+.filter-question .btn-gray:first-child {
+    border-bottom-left-radius: 0.2em;
+    border-top-left-radius: 0.2em;
+}
+
+.filter-question .btn-gray:nth-last-child(2) {
+    border-bottom-right-radius: 0.2em;
+    border-top-right-radius: 0.2em;
+}
```

Comment:
Let's refactor to use `button` instead of `.btn-gray` in these rules. Should work, and it allows us to change the variant of the button without messing the styles here.